db/dev_seeds/sdg.rb

```diff
@@ -1,3 +1,35 @@
 section "Creating Sustainable Development Goals" do
   load Rails.root.join("db", "sdg.rb")
+
+  SDG::Target.sample(30).each do |target|
+    title = "Title for default locale"
+    description = "Description for default locale"
+    rand(2..3).times do |n|
+      local_target = SDG::LocalTarget.create!(code: "#{target.code}.#{n}",
+                                              title: title,
+                                              description: description,
+                                              target: target)
+      random_locales.map do |locale|
+        Globalize.with_locale(locale) do
+          local_target.title = "Title for locale #{locale}"
+          local_target.description = "Description for locale #{locale}"
+          local_target.save!
+        end
+      end
+    end
+  end
+
+  relatables = [Debate.last, Proposal.last, Poll.last, Legislation::Process.last, Budget::Investment.last]
+  relatables.each do |relatable|
+    3.times do
+      random_goal = SDG::Goal.sample
+      random_target = random_goal.targets.sample
+      random_local_target = random_target.local_targets.sample
+      relatable.sdg_goals << random_goal if [true, false].sample
+      relatable.sdg_targets << random_target if [true, false].sample
```

Comment:
I think we have to add the target only if the goal has been added :thinking:, since right now the model doesn't assign the goal automatically when a target is added.

So how about:

```ruby
    Array(SDG::Goal.sample(rand(1..3))).each do |goal|
      target = goal.targets.sample
      local_target = target.local_targets.sample
      relatable.sdg_goals << goal
      relatable.sdg_targets << target if [true, false].sample
      # ...
```

Reply:
On the other hand, AFAIK assigning a local target when a target has not been assigned might be fine :thinking:, although I'm not sure.

Reply:
Sounds reasonable! Probably it is too soon to make the final choice; maybe the customer changes his mind when we have the form interface, and we can test the user's experience. In the meantime, I will store all redundant relations if you agree.

Reply:
Fine with me :ok_hand:.